src/dcd_mapping/cli.py

```diff
@@ -25,6 +25,14 @@
     default=False,
     help="Enable debug logging",
 )
+@click.option(
+    "--hide",
+    "-h",
+    is_flag=True,
+    show_default=True,
```

Comment:
niche: you don't need to include this, it doesn't do anything if `default=False`